packages/core/src/__tests__/login.spec.ts

```diff
@@ -138,32 +136,6 @@ describe('Login', () => {
       },
     };
 
-    test('successful getIdInfoFromApiKey', async () => {
```

Comment:
Why are we removing this?

Reply:
it's never used? as you can see I removed it and nothing happened

Reply:
so you are sure no one of our users uses it 🤔 ?

Reply:
it wasn't exported from the main module and it was hidden from ref docs

Reply:
idk why we have it

Reply:
but less code is always better)